preupg/cli.py

```diff
@@ -54,126 +54,166 @@ def __init__(self, args=None):
+            help="Generate plain text assessment report alongside XML and HTML"
+                 " reports. The text report is converted from HTML using"
+                 " elinks, lynx or w3m tool."
         )
         self.parser.add_option(
-            "-v",
-            "--verbose",
+            "-v", "--verbose",
             action="count",
             default=0,
-            help="Verbose mode"
+            help="Show more information during the assessment."
         )
         self.parser.add_option(
-            "--cleanup",
+            "-C", "--cleanup",
             action="store_true",
             default=False,
-            help="Clean data created by preupgrade-assistant"
+            help="Remove all the files created by previous runs of Preupgrade"
+                 " Assistant."
         )
         self.parser.add_option(
-            "-m",
-            "--mode",
+            "-m", "--mode",
             metavar="MODE",
             choices=['migrate', 'upgrade'],
-            help="Select mode which can be used for migration or upgrade"
+            help="Select what you plan to do with the system after performing"
+                 " its assessment by Preupgrate Assistant - migration or"
+                 " upgrade. Both modes are selected by default. This option"
+                 " may only affect behaviour of the modules - they can provide"
+                 " different results when only one mode is selected. Use one"
+                 " of these values: migrate, upgrade. It may be that modules"
+                 " behave the same no matter what mode is selected."
         )
         self.parser.add_option(
-            "--kickstart",
+            "-k", "--kickstart",
             action="store_true",
             default=False,
-            help="Generate kickstart"
+            help="Generate kickstart file that is to be used for migration."
+                 " Preupgrade Assistant system assessment needs to be"
+                 " performed before using this option. You may use the"
+                 " preupg-kickstart-generator tool instead of this option -"
+                 " it does the same."
         )
         self.parser.add_option(
             "--select-rules",
             metavar="RULES",
-            help="Run a set of contents give as a comma-separated list."
+            help="Execute just a subset of modules out of a module set."
+                 " Multiple modules are to be separated by a comma." +
+                 " " * 5 +
+                 "Example:  --select-rules xccdf_preupg_rule_networking_vsftpd"
+                 "_check,xccdf_preupg_rule_networking_bind_configuration_check"
         )
         self.parser.add_option(
-            "--list-rules",
+            "-L", "--list-rules",
             action="store_true",
             default=False,
-            help="List available contents"
+            help="List all the modules available within a module set."
```

Comment:
Really it will be in the one module set or for all module sets? Should be then specified/mentioned use of --scan or -c to choose correct module set if there exists more modules? (in case it will print all dinf rules of all modules sets, it will not be probably required)